angular/src/styles/variables/_colors.scss

```diff
@@ -0,0 +1,12 @@
+$white: #FFF;
+$black: #000;
+$red: #F00;
+$blue: #00F;
+$gray: #A8A8A8;
+$wx-blue: #00ADEE;
+$wx-green: #BED23C;
+$bg-gray: #F4F4F4;
+$book-gray: #828282;
+$bg-popup: rgba(0, 0, 0, 0.5);
+$line-popup: #C8C8C8;
+$box-shadow-navbar:rgba(0, 0, 0, 0.5);
```

Comment:
Use this [web](https://chir.ag/projects/name-that-color/#6195ED) for naming colors